### DISCUSSION TITLE
az-devops-like-a-boss/

### DISCUSSION THREAD

**Discussion by giscus** [colindembovsky/colindembovsky.github.io#18] [Giscussions]
# az-devops-like-a-boss/

One of the best features of Azure DevOps is the extensive API. However, while having a REST API is great, interacting with a service at HTTP level can be frustrating. In this post, I examine the az devops cli using 10 practical examples.

https://colinsalmcorner.com/az-devops-like-a-boss/

**Comment by shuft** [comment]
Thanks so much for the writeup here! Helped me get my head round the `--query` option, which was the missing bit for making az repos cli useful to me!
